addon/wolfcrypt/rsa.cpp

```diff
@@ -0,0 +1,130 @@
+#include "./h/rsa.h"
+
+Napi::Number sizeof_RsaKey(const Napi::CallbackInfo& info)
+{
+  Napi::Env env = info.Env();
+
+  return Napi::Number::New( env, sizeof( RsaKey ) );
+}
+
+Napi::Number bind_wc_RsaEncryptSize(const Napi::CallbackInfo& info)
+{
+  int ret;
+  Napi::Env env = info.Env();
+  RsaKey* rsa = (RsaKey*)( info[0].As<Napi::Uint8Array>().Data() );
+
+  ret = wc_RsaEncryptSize( rsa );
+
+  return Napi::Number::New( env, ret );
+}
+
+Napi::Number bind_wc_InitRsaKey(const Napi::CallbackInfo& info)
+{
+  int ret;
+  Napi::Env env = info.Env();
+  RsaKey* rsa = (RsaKey*)( info[0].As<Napi::Uint8Array>().Data() );
+
+  ret = wc_InitRsaKey( rsa, NULL );
+
+  rsa->rng = (WC_RNG*)XMALLOC( sizeof( WC_RNG ), rsa->heap, DYNAMIC_TYPE_RNG );
```

Comment:
Consider using `wc_rng_new` and `wc_rng_free`. Those handle the init also and come from heap.